src/app/modules/shared-components/components/add-content/add-content.component.ts

```diff
@@ -25,33 +22,47 @@ export class AddContentComponent implements OnInit, OnDestroy {
 
   newContentForm: FormGroup = this.formBuilder.group(defaultFormValues);
   trimmedInputsValue = defaultFormValues;
+  itemsFound: ItemFound[] = [];
 
-  focused?: 'searchExisting' | 'create';
-  itemsFound: ItemFound[] = [
-    { title: 'test', type: 'Chapter' },
-    { title: 'test', type: 'Chapter' },
-    { title: 'test', type: 'Chapter' },
-    { title: 'test', type: 'Chapter' },
-  ];
+  state: 'loading' | 'loaded' = 'loading';
+  focused?: 'create' | 'searchExisting';
 
-  private subscription?: Subscription;
+  private subscriptions: Subscription[] = [];
 
   constructor(
     private formBuilder: FormBuilder,
+    private searchItemService: SearchItemService,
   ) {
   }
 
   ngOnInit(): void {
-    this.subscription = this.newContentForm.valueChanges.subscribe((changes: typeof defaultFormValues) => {
-      this.trimmedInputsValue = {
-        create: changes.create.trim(),
-        searchExisting: changes.searchExisting.trim(),
-      };
-    });
+    this.subscriptions.push(
+      this.newContentForm.valueChanges.subscribe((changes: typeof defaultFormValues) => {
+        this.trimmedInputsValue = {
+          create: changes.create.trim(),
+          searchExisting: changes.searchExisting.trim(),
+        };
+      })
+    );
+
+    const existingTitleControl: Observable<string> | undefined = this.newContentForm.get('searchExisting')?.valueChanges;
```

Comment:
"newContentForm" was not supposed to create new content at first? (vs addContent)

Reply:
I've renamed the form.